### PR TITLE
Update posit_team_applications_faq.md

### PR DESCRIPTION
**Pull request by terrymclaughlin**
Explain that there are no plans to support Java on Posit Workbench.

### CODE REVIEW

**Comment by terrymclaughlin** [comment]
I don't know if we need more detail with this one e.g. other packages that you are aware of that rely on Java.  Or do we need to revisit supporting Java on Posit Workbench (perhaps a longer term plan?)

**Comment by jakeybob** [comment]
Might want to mention `RJDBC` as well, as I think this was mentioned somewhere as being the officially recommended database connection package for the CDW (and I assume it depends on java at some point). I don't think I've ever seen anyone use it, because the usual/existing solutions work much better, but someone somewhere might.

**Comment by jakeybob** [comment]
In terms of whether Java should be supported or not, I'd lean towards "not if we can possibly avoid it". As it is, the two main use cases I can see (interacting with Excel files, and using RJDBC) both have non-java-dependant alternatives which are very widely used (so have lots of existing code examples) and are also just *better*.

If a team has a *lot* of existing code that uses one of these packages, then that gets tricky -- but even then I think a strong encouragement to get off the Java train would be wise (maybe it could be supported for ~6 months... although I don't know if that ends up being much less work than supporting indefinitely?)

**Review by rmccreath — COMMENTED** [review]
While I agree with the general consensus of not pursuing the support of Java, should we explore other packages that rely on this before making a definitive call? Also, I don't believe there should be any projects that are reliant on `{xlsx}` as there were existing problems and `{openxlsx}` has been the recommended solution. Finally, although not useful at this point, this should really have been caught in UAT?

**Comment by terrymclaughlin** [comment]
Here is the list of all R packages that import, depend on or suggests `{rJava}`.

```
Andromeda
AntAngioCOOL
arulesCBA
arulesNBMiner
AWR
AWR.Athena
AWR.Kinesis
AWR.KMS
bartMachine
bartMachineJARs
BASiNET
BEACH
beastier
BeastJar
boilerpipeR
CADStat
CBSr
ChoR
collUtils
CommonJavaJars
commonsMath
corehunter
coreNLP
DatabaseConnector
DatabaseConnectorJars
DecorateR
Deducer
dialr
dialrjars
dismo
dsa
ecospat
ENMeval
EvaluateCore
EvidenceSynthesis
fsdaR
G2Sd
glmulti
gMCP
GreedyExperimentalDesign
GreedyExperimentalDesignJARs
helloJavaWorld
hive
implyr
InSilicoVA
iplots
ISM
JavaGD
JGR
joinXL
llama
mailR
mallet
matchingMarkets
MLMOI
mutossGUI
mwa
NicheBarcoding
openNLP
openNLPdata
OpenStreetMap
partykit
pguIMP
phyloregion
PortfolioEffectEstim
PortfolioEffectHFT
prcbench
predicts
qCBA
r5r
rapidraker
Rbgs
rCBA
rcdk
rcdklibs
rChoiceDialogs
rCMA
Rdroolsjars
REPPlab
restoptr
rflsgen
rGroovy
RH2
RJDBC
RJDemetra
rJPSGCS
RJSDMX
rkafka
rkafkajars
RKEA
RKEAjars
RKEEL
rmcfs
RMOA
RMOAjars
RNCBIEUtilsLibs
RNetLogo
rrepast
RSCAT
RSP
rsubgroup
rviewgraph
RWeka
RWekajars
RxnSim
scagnostics
scatteR
sdmApp
SDMPlay
SDMtune
sdpdth
sjdbc
spcosa
SqlRender
staplr
stream
streamMOA
subspace
svMisc
venneuler
wallace
wordnet
XLConnect
xlsx
xlsxjars
zooimage
```

Do you think we need a poll on the R User Group to see how many of these packages are in use?

**Comment by terrymclaughlin** [comment]
@jakeybob @CliveWG @ciarag01 

Would any of you be able to review this pull request so we can get the information about Java on Posit Workbench out there to link to?  This question is coming up more often now.

Thanks in advance!

File: posit_team_applications_faq.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ You can then install the `{phsmethods}` package and all of its other dependencie
 install.packages("phsmethods")
 ```
 
+#### I can't install / use a package because it needs `{rJava}`
+
+Some packages, e.g. `{xlsx}`, `{XLconnect}`, depend on an installation of [Java](https://en.wikipedia.org/wiki/Java_(software_platform)) in order to work.  There are no plans to support Java on Posit Workbench.  Alternative packages such as `{openxlsx}` that do not rely on Java can be used instead.
```
Comment (CliveWG) [inline posit_team_applications_faq.md:37]: ....... no current or future plans to support Java on Posit Workbench.  Alternative packages such as `{openxlsx}` that do not rely on Java should be used instead.

**Review by jakeybob — APPROVED** [review]

File: posit_team_applications_faq.md (hunk 1 above)
Comment (terrymclaughlin) [reply posit_team_applications_faq.md]: @CliveWG I've updated as per your suggested wording.  Thanks!

**Pull request merged**